extendables/checkChatForImage.js

```diff
@@ -0,0 +1,27 @@
+const { Extendable } = require('klasa');
+
+module.exports = class extends Extendable {
+
+	constructor(...args) {
+		super(...args, {
+			appliesTo: ['KlasaMessage'],
+			klasa: true
+		});
+	}
+
+	async extend(msg) {
```

Comment:
Remove msg, you're not going to do `msg.checkChatForImage(msg);`. Also for naming conventions (and now that I realise it), you may want to call it `msg.channel.fetchImage()`, since:

1. This method belongs to a channel, it does nothing to the message but to the channel.
1. It fetches, therefore fetchImage is more appropiate.